### PR TITLE
ADEN-11841 Integrate new Pre-Bidder: Roundel

### PR DESCRIPTION
**Pull request by szymon-nawrocki**
### Ticket
https://fandom.atlassian.net/browse/ADEN-11841

### CODE REVIEW

File: platforms/ucp-desktop/setup/context/prebid/ucp-desktop-prebid-config.setup.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { getTripleliftContext } from '../../../bidders/prebid/triplelift';
 import { getVerizonContext } from '../../../bidders/prebid/verizon';
 import { getWikiaContext } from '../../../bidders/prebid/wikia';
 import { getWikiaVideoContext } from '../../../bidders/prebid/wikia-video';
+import { getRoundelContext } from '../../../bidders/prebid/roundel';
```
Comment (dmnsobczak) [inline platforms/ucp-desktop/setup/context/prebid/ucp-desktop-prebid-config.setup.ts:22]: Alphabetical order

File: src/ad-bidders/prebid/adapters/roundel.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import { EXTENDED_MAX_CPM, PrebidAdapter } from '../prebid-adapter';
+import { PrebidAdSlotConfig } from '../prebid-models';
+import { context } from '@ad-engine/core';
+
+export class Roundel extends PrebidAdapter {
+	static bidderName = 'roundel';
+	aliases = {
+		indexExchange: [Roundel.bidderName],
+	};
+	maxCpm = EXTENDED_MAX_CPM;
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/adapters/roundel.ts:10]: Not needed, let's keep standard CPM (up do 20$)

File: src/ad-bidders/prebid/adapters/roundel.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import { EXTENDED_MAX_CPM, PrebidAdapter } from '../prebid-adapter';
+import { PrebidAdSlotConfig } from '../prebid-models';
+import { context } from '@ad-engine/core';
+
+export class Roundel extends PrebidAdapter {
+	static bidderName = 'roundel';
+	aliases = {
+		indexExchange: [Roundel.bidderName],
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/adapters/roundel.ts:8]: Should be:
`ix: [Roundel.bidderName],`

File: platforms/ucp-desktop/setup/context/prebid/ucp-desktop-prebid-config.setup.ts (hunk 1 above)
Comment (szymon-nawrocki) [reply platforms/ucp-desktop/setup/context/prebid/ucp-desktop-prebid-config.setup.ts]: Done ✅

File: src/ad-bidders/prebid/adapters/roundel.ts (hunk 2 above)
Comment (szymon-nawrocki) [reply src/ad-bidders/prebid/adapters/roundel.ts]: Done ✅

File: src/ad-bidders/prebid/adapters/roundel.ts (hunk 3 above)
Comment (szymon-nawrocki) [reply src/ad-bidders/prebid/adapters/roundel.ts]: Done ✅

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**